crates/sui-framework/sources/governance/voting_power.move

```diff
@@ -57,7 +57,8 @@ module sui::voting_power {
         validators: &vector<Validator>,
         threshold: u64,
     ): (vector<VotingPowerInfo>, u64) {
-        let total_stake = total_stake(validators);
+        // avoid divide by zero below in the case where total stake is 0
+        let total_stake = math::max(total_stake(validators), 1);
```

Comment:
You probably also need to account for an individual validators stake being zero as well right? otherwise we'll end up with validators with 0 voting power.

Reply:
ah on closer inspection what you have here is sufficient